jupyterhub/apihandlers/base.py

```diff
@@ -131,9 +132,16 @@ def write_error(self, status_code, **kwargs):
             json.dumps({'status': status_code, 'message': message or status_message})
         )
 
-    def server_model(self, spawner, include_state=False):
+    def server_model(self, spawner, include_state=False, include_user_and_id=False):
         """Get the JSON model for a Spawner"""
-        return {
+        if isinstance(spawner, orm.Spawner):
+            # TODO(mriedem): Would be nice to join the user field from
+            # the Spawner ORM object when we fetch it from the DB so we don't
+            # need the roundtrip.
+            orm_user = orm.User.find_by_id(self.db, spawner.user_id)
+            user = self._user_from_orm(orm_user)
+            spawner = spawner_obj.Spawner(orm_spawner=spawner, user=user)
+        server = {
             'name': spawner.name,
             'last_activity': isoformat(spawner.orm_spawner.last_activity),
             'started': isoformat(spawner.orm_spawner.started),
```

Comment:
Note that with the `Spawner.__getattr__` override now we don't need to touch the `orm_spawner` directly here. Future cleanup I guess.